ckcp/gitops.sh

```diff
@@ -113,14 +119,14 @@ install_openshift_gitops() {
   # Install the gitops operator
   #############################################################################
   echo -n "  - OpenShift-GitOps: "
-  oc apply -f "$GITOPS_DIR/$APP.yaml" --wait >/dev/null
+  kubectl apply -f "$GITOPS_DIR/$APP.yaml" --wait >/dev/null
   echo "OK"
 
   #############################################################################
   # Wait for the URL to be available
   #############################################################################
   echo -n "  - ArgoCD dashboard: "
-  test_cmd="oc get route/openshift-gitops-server --ignore-not-found -n $APP -o jsonpath={.spec.host}"
+  test_cmd="kubectl get route/openshift-gitops-server --ignore-not-found -n $APP -o jsonpath={.spec.host}"
```

Comment:
If it matters: route and openshift-gitops-server are OpenShift specific

Reply:
Same as above, I removed all of Kubernetes specific, for now this script just support Openshift. I will add Kubernetes support in a separate script.